data/hai/hai reveal 1 intro.txt

```diff
@@ -670,16 +692,13 @@ mission "Hai Leaks Response 2C"
 			label met
 			action
 				log "People" "Samuel Remington" `Remington is being sent as the representative of the Syndicate to help deal with the situation regarding the Hai.`
-			`	"Captain <last>, is it? It's been a good while since we last met," he says while shaking your hand, perhaps a little too firmly. "You do remember me, right? I don't want this to be an awkward situation where only I..." He stops himself. "You know what, nevermind. We're not here to make small talk. We have bigger things to worry about. I'm looking forward to working alongside you to handle this emerging crisis.`
+			`	"Captain <last>, is it? It's been a good while since we last met," he says while shaking your hand, perhaps a little too firmly. He gestures to the woman behind him and says, "Let me introduce you to Terry."`
 			label next
-			`	"Now, allow me to introduce you to Terry," Remington says while gesturing to the woman behind him.`
 			`	"Terry Adrianopoulos. Pleasure to meet you," she says with a broad smile and a slight bow. The young woman goes from calculating to gregarious in the blink of an eye as your attention shifts to her. She seems at once pleasant and approachable while at the same time being slightly off-putting and uncanny. It's an eerie feeling.`
-			`	"Terry will be accompanying me as... well, let's call her my secretary," Remington says with a sly smile. Terry seems none too pleased to hear that, but she doesn't correct him. "I have men who can hold the fort here while I'm gone, and I really do need to get some fresh air, so the two of us will be joining you. There is an internal political situation going on within the Syndicate that has been causing me quite the headache recently," he says. You think that he may have glanced at Terry as he said that, but you aren't sure if it was just incidental or indicative of something. "Nothing I can discuss in detail unfortunately, you know, with all the NDAs involved, but something I'll certainly enjoy taking a break from.`
+			`	"Terry will be accompanying me as... well, let's call her my secretary," Remington says with a sly smile. Terry seems none too pleased to hear that, but she doesn't correct him. "I have men who can hold the fort here while I'm gone, so the two of us will be joining you. There is an internal political situation going on within the Syndicate that has been causing me quite the headache recently." You catch him take a furtive glance at Terry as he says that, but Remington presses on immediately. "Nothing I can discuss in detail, you know, with all the NDAs involved, but it's something I'll certainly enjoy taking a break from.`
```

Comment:
```suggestion
			`	"Terry will be accompanying me as... well, let's call her my secretary," Remington says with a sly smile. Terry seems none too pleased to hear that, but she doesn't correct him. "I have men who can hold the fort here while I'm gone, and I really do need to get some fresh air, so the two of us will be joining you. There is an internal political situation going on within the Syndicate that has been causing me quite the headache recently." You catch him take a furtive glance at Terry as he says that, but Remington presses on immediately. "Nothing I can discuss in detail, you know, with all the NDAs involved, but it's something I'll certainly enjoy taking a break from.`
```
I like the fresh air bit but I didn't like where it was. But I can see how the next bit is left hanging.